runners/flink/src/test/java/org/apache/beam/runners/flink/PortableTimersExecutionTest.java

```diff
@@ -185,11 +178,10 @@ public void processElement(ProcessContext context) {
             Collections.emptyList());
 
     jobInvocation.start();
-    long timeout = System.currentTimeMillis() + 60 * 1000;
+    long timeout = System.currentTimeMillis() + 2 * 60 * 1000;
     while (jobInvocation.getState() != Enum.DONE && System.currentTimeMillis() < timeout) {
       Thread.sleep(1000);
     }
     assertThat(jobInvocation.getState(), is(Enum.DONE));
```

Comment:
We can also remove this line if we remove the timeout because the while loop will only return when it is done.

Reply:
Done